examples/README.md

```diff
@@ -1,7 +1,23 @@
 # IBM Analog Hardware Acceleration Kit: Examples
 
 We have many different examples to explore the many features of the IBM Analog Hardware
-Acceleration Kit:
+Acceleration Kit.
+
+All the tests do not have a main.py file. We use Pytest for unit testing, which does not need any main.py file. A number of common developmental tools can be installed with pip install -r requirements-dev.txt
```

Comment:
Suggestion:

"This project contains a [unittest](https://docs.python.org/3/library/unittest.html) compatible test suite, that can be executed by any Python test runner. The recommended runner is `pytest`, which can be installed along with a number of other development tools used in `aihwkit` via:

```
pip install -r requirements-dev.txt 
```
"

The thinking is that the link would provide more material and information for users, including explanation about the structure and the lack of `main.py` - and we emphasize that the test can be run with other runners (but we favor `pytest`).